package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mock-yeah",
-  "version": "0.5.0",
+  "version": "0.7.0",
```

Comment:
There is weirdness here too. The package named changed and the version is now `0.11.0`.